scrapper.py

```diff
@@ -26,36 +31,71 @@ class Crawler:
     Crawler implementation
     """
     def __init__(self, seed_urls, max_articles: int):
-        pass
+        self.seed_urls = seed_urls
+        self.max_articles = max_articles
+        self.urls = []
+
 
     def _extract_url(self, article_bs):
-        pass
+        class_bs = article_bs.find('div', class_='view-content view-rows')
+        title_bs = class_bs.find_all('td', class_="views-field views-field-title table__cell")
+        for link in title_bs:
+            link = link.find('a')
+            self.urls.append('https://rjano.ruslang.ru' + link['href'])
+
 
     def find_articles(self):
         """
         Finds articles
         """
-        pass
+        headers = {'user-agent': 'Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) '
+                                 'AppleWebKit/537.36 (KHTML, like Gecko) Chrome/98.0.4758.102 Safari/537.36'}
+        for seed_url in self.seed_urls:
+            response = requests.get(seed_url, headers)
+            article_bs = BeautifulSoup(response.text, 'html.parser')
+            self._extract_url(article_bs)
+
 
     def get_search_urls(self):
         """
         Returns seed_urls param
         """
-        pass
+        return self.seed_urls
 
 
 def prepare_environment(base_path):
     """
     Creates ASSETS_PATH folder if not created and removes existing folder
     """
-    pass
+    try:
+        os.rmdir(base_path)
+    except FileNotFoundError:
+        pass
+    os.mkdir(base_path)
 
 
 def validate_config(crawler_path):
     """
     Validates given config
     """
-    pass
+    with open(crawler_path, 'r', encoding='utf-8') as file:
+        config = json.load(file)
+    seed_urls = config['seed_urls']
+    max_articles = config['total_articles_to_find_and_parse']
+    for seed_url in seed_urls:
+        if seed_url[0:8] != 'https://' and seed_url[0:7] != 'http://':
+            raise IncorrectURLError
+    if not seed_urls:
+        raise IncorrectURLError
+    if not isinstance(max_articles, int):
+        raise IncorrectNumberOfArticlesError
+    if max_articles > 100:
+        raise NumberOfArticlesOutOfRangeError
+    if max_articles == 0 or max_articles < 0:
+        raise IncorrectNumberOfArticlesError
+    if not isinstance(seed_urls, list):
+        raise IncorrectURLError
+    return seed_urls, max_articles
```

Comment:
start implementing HTMLParser in order to get tests checking this method. at least create an empty class, otherwise tests will never start